bot/cogs/grades_data/grades_cog.py

```diff
@@ -249,12 +277,12 @@ async def grades(self, ctx, course, year=2014):
 
         except NotADirectoryError: # output if course doesn't exist
             embed = discord.Embed(title="Grades", color=Colors.Error)
-            result = 'That\'s not a course\n Are you sure you used the proper notation (ex: cpsc-2120)'
+            result = 'That\'s not a course\n Are you sure you used the proper notation (ex: cpsc-2120)?'
             embed.add_field(name="ERROR: Course doesn't exist", value=result, inline=False)
         
-        except FileNotFoundError:
+        except FileNotFoundError as e:
             embed = discord.Embed(title="Grades", color=Colors.Error)
-            result = "Either master.json or master_prof.json are not found in the proper directory"
+            result = f'Files `{e}` are not in directory. Reminder that the files go from 2014-2019!'
```

Comment:
Thought: if we make the previously mentioned iterable of strings for the load function an instance member we could dynamically create this string based off that, example 

`'f'Files {e} are not in directory. Reminder that the files go from {self.years[0]}-{self.years[-1]}!'`